src/include/cephfs/libcephfs.h

```diff
@@ -1947,6 +1950,8 @@ int64_t ceph_ll_readv(struct ceph_mount_info *cmount, struct Fh *fh,
 		      const struct iovec *iov, int iovcnt, int64_t off);
 int64_t ceph_ll_writev(struct ceph_mount_info *cmount, struct Fh *fh,
 		       const struct iovec *iov, int iovcnt, int64_t off);
+void ceph_ll_readv_writev(struct ceph_mount_info *cmount,
```

Comment:
This is a new interface only being used by Ganesha right now, but we still need to add _a_ test to avoid future breakages due to changes in other parts in the client.

Reply:
... and now I see `ceph_ll_nonblocking_readv_writev` also potentially using zero-copy.